src/web/app/components/question-edit-form/question-edit-form.component.ts

```diff
@@ -15,7 +15,26 @@ import { SimpleModalType } from '../simple-modal/simple-modal-type';
 import { collapseAnim } from '../teammates-common/collapse-anim';
 import { QuestionEditFormMode, QuestionEditFormModel } from './question-edit-form-model';
 
-const CLEAN_PROPERTIES: Set<string> = new Set<string>(['isEditable', 'isCollapsed', 'isChanged']);
+const FEEDBACK_PATH_PROPERTIES: Set<string> = new Set<string>([
```

Comment:
We can use a more strict type here (and similarly below)
```suggestion
const FEEDBACK_PATH_PROPERTIES: Set<keyof QuestionEditFormModel> = new Set<string>([
```

Reply:
I think this change would not be compatible with the way we use the keys below,
`Object.keys(obj).some((key: string) => VISIBILITY_PROPERTIES.has(key))`
Where `key` is expected to be of type `string`, and not `keyof QuestionEditFormModel`? Maybe it's possible but I can't seem to get it to work.
We can do this to ensure the keys added in do not have typos tho, do you want me to proceed with this change?
`const FEEDBACK_PATH_PROPERTIES: Set<string> = new Set<keyof QuestionEditFormModel>([...`

Reply:
It's fine if it is difficult to do so